apps/drupal-default/particle_theme/templates/layout/header/custom-header.html.twig

```diff
@@ -15,21 +33,17 @@
         <div class="header__main__right">
           {{ page.header_secondary_menu }}
           {% include '@themag/base-layout/header/components/user-action-menu.html.twig' %}
+          <div{{ attributes.addClass(classes) }}>
```

Comment:
I moved the social media icons next to the user-actions menu. Seems like it visually makes sense since icons are used.